GIT-PROCESS.md

```diff
@@ -0,0 +1,146 @@
+# Colorado Care Exchange Git Process Guide
+
+Our master branch is what gets deployed to production automatically.
+
+**CAUTION** Please be careful and only push to master once QA and PRODUCT are ready for the deployment
+
+# Branches
+
+## Master
+
+The master branch at origin should be familiar to every Git user. We consider origin/master to be the main branch where the source code of HEAD always reflects a production-ready state. The master branch is set to automatically deploy to production. 
+
+## Acceptance
+
+The acceptance branch automatically gets deployed to the acceptance environment. The product team uses this branch as a final verification to vet the next release candidate.
+
+## Canary
+
+The canary branch automatically gets deployed to the canary (dev) instance. This where developers push their code complete feature/bug/hotfix/feature branches. The reason for the name canary relates to the proverbial canary in the coal mine. In practice, the team member merges its project branches into the canary branch to make sure that their branches “play nice” with the rest of the team’s project branches and the product as a whole. The QA team tests each ticket against the canary branch after the developer merged their ticket branch into canary.
+
+
+## Release Candidates
+
+The release candidate branches are used by the product team to acceptance test the larger feature enhancement projects. The product team may decide that a few features should be held up until the necessary documentation has been created and communicated to the userbase. Once the materials are complete, the formal release candidate can be released to production.
+
+Release candidate branches are first based on master. The QA team will test feature branches on the canary branch. Once these feature branches are approved by the QA team, the developer of the feature branches will merge then into the current release candidate branch. The product team will review the release candidate one the release candidate is merged into acceptance. If issues are found on the current release candidate, the product team can cancel the release candidate if the issue(s) is deemed severe or they can allow the developer to fix any issues that are discovered. Once the product team accepts the new release candidate, it will be scheduled for deployment.
```

Comment:
I thought the workflow was canary -> acceptance -> rc (if it passes acceptance). I think this proposal might make it difficult to back out of a deploying a feature

If so, there are other spots on this doc that need to be updated as well